### ISSUE TITLE
Make event title and image appear when events are shared on FB

### ISSUE THREAD

**Issue by InciteDemocracy** [ResistanceCalendar/resistance-calendar-frontend#177]
Currently FB share window looks like this: 

![screenshot 2017-06-19 08 50 17](https://user-images.githubusercontent.com/25624431/27300736-643b15b4-54cc-11e7-9a77-196ef73e1bc9.png)

**Comment by pdw207** [comment]
@sdotson Sounds like you might have the most experience with this. Does this relate to an app id or a regression since your PR on social buttons was merged `https://github.com/ResistanceCalendar/resistance-calendar-frontend/pull/101`

**Comment by sdotson** [comment]
Bummer. It looks like Facebook deprecated picture, caption, and description as things we can pass to the share dialogue api: https://developers.facebook.com/docs/apps/changelog#v2_9_90_day_deprecations

Looks like they really want folks to shift to open-graph tags, which has to be implemented server side.

Their API now only lets us specify a quote, url, and hashtag:
https://developers.facebook.com/docs/sharing/reference/share-dialog

**Comment by pdw207** [comment]
An argument for an isomorphic app.

**Comment by dorono** [comment]
Assuming that FB's policy of only allowing the API to give picture, caption, etc via the back end, we could always go with something like https://www.sharethis.com. It generates a pop-up window (as opposed to a modal overlay), but it seems to get the job done on desktop as well as mobile, so perhaps it's more important to make the content sharable than to make things look extra-slick.

**Comment by dorono** [comment]
Adding Twitter to this issue, since we're also not getting images from Twitter shares. Again, ShareThis would solve this issue with minimal work.

**Comment by dorono** [comment]
Though React Helmet would work, but that only works for server-side rendered apps. This bit of code might work: https://www.reddit.com/r/reactjs/comments/5j7100/react_dynamic_facebook_sharing_dialogue_question/

**Comment by albjeremias** [comment]
u just need to implement the correct meta-tags, on your html!

https://moz.com/blog/meta-data-templates-123

**Comment by dorono** [comment]
@albjeremias I don't believe that it would address the issue with the Facebook crawler not being able to find previews of a given page from the website. I think those meta tags help for sharing, which I think is working ok now, but it's for when someone goes into Facebook.com or Twitter.com and attempts to paste a URL from Resistance Calendar. The FB and Twitter crawlers do not execute JavaScript, so there is no way for them to see the JS-generated page content, so there has to be static HTML version of each page (which the end user never sees, of course).

Does that make sense?

**Comment by albjeremias** [comment]
no. and if you don't believe what can I say? good luck
